### PR TITLE
Make the url in UI info box clickable

### PR DESCRIPTION
**Pull request by tjisanaMS**
## Bug / Requirement Description
Clearly and concisely describe the problem.

## Solution description
Describe your code changes in detail for reviewers.

## Checklist:
- [x] Test
- [ ] Example (both test_plan.py and .rst)
- [ ] Documentation (API)
- [x] News fragment present for release notes
- [x] MS info leakage check
- [ ] For new driver: driver index page
- [ ] For new assertion: ui/pdf/std renderers, documentation
- [ ] For new cmdline arg: documentation

### CODE REVIEW

File: testplan/web_ui/testing/src/Toolbar/Toolbar.js [hunk 1]
```diff
@@ -503,10 +504,24 @@ const getToggledButtonStyle = (status) => {
...
   if (!report || !report.information) {
     return "No information to display.";
   }
   const infoList = report.information.map((item, i) => {
+    const linkifyIgnore = ['user', 'command_line_string', 'python_version', 'hostname', 'start', 'end']
+    if (!linkifyIgnore.includes(item[0])) {
+      return (
+        <tr key={i}>
+          <td className={css(styles.infoTableKey)}>{item[0]}</td>
+          <td className={css(styles.infoTableValue)}><div dangerouslySetInnerHTML={{__html:linkifyUrls(
```
Comment (kn-ms) [inline testplan/web_ui/testing/src/Toolbar/Toolbar.js:518]: There is too many duplication here with the ignored case.

I see two option 

1. create a function that take an item, and do the check and ither return the div with linkifyUrls or item[0]
2. use the ternary operator inline

File: testplan/web_ui/testing/src/Toolbar/Toolbar.js [hunk 2]
```diff
@@ -503,14 +505,37 @@ const getToggledButtonStyle = (status) => {
...
+
   if (!report || !report.information) {
     return "No information to display.";
   }
   const infoList = report.information.map((item, i) => {
+    const linkifyIgnore = ['user', 'command_line_string',
+    'python_version', 'hostname', 'start', 'end'];
+
+
+    let columnData = !linkifyIgnore.includes(item[0]) ? < td className = {
```
Comment (kn-ms) [inline testplan/web_ui/testing/src/Toolbar/Toolbar.js:518]: probably an exact match would be better instead of includes

Comment (kn-ms) [reply testplan/web_ui/testing/src/Toolbar/Toolbar.js:518]: The ternary operator is hard to read here, and you repeating the same <td>.

a couple of options you can put the content goes into the <td> into a variable (here you can use the ternary operator )which is either the <div> or just the string content. Alternatively the working out of the linkify thing can be moved to it's own method together with the list of ignored fields, and just use that method in the template.

File: testplan/web_ui/testing/src/Toolbar/Toolbar.js (hunk 1 above)
Comment (tjisanaMS) [reply testplan/web_ui/testing/src/Toolbar/Toolbar.js]: Done

File: testplan/web_ui/testing/src/Toolbar/Toolbar.js (hunk 2 above)
Comment (tjisanaMS) [reply testplan/web_ui/testing/src/Toolbar/Toolbar.js]: Done

**Review by kn-ms — APPROVED** [review]

**Review by Pyifan — APPROVED** [review]

**Pull request merged**